managed/models/service_model.go

```diff
@@ -99,6 +106,33 @@ func (s *Service) SetCustomLabels(m map[string]string) error {
 	return setLabels(m, &s.CustomLabels)
 }
 
+// ChangeStandardLabels changes standard labels and returns a list of columns which were updated.
+func (s *Service) ChangeStandardLabels(labels ServiceStandardLabelsParams) []string {
+	columns := []string{}
+
+	if labels.Cluster != nil {
+		columns = append(columns, "cluster")
+		s.Cluster = *labels.Cluster
+	}
+
+	if labels.Environment != nil {
+		columns = append(columns, "environment")
+		s.Environment = *labels.Environment
+	}
+
+	if labels.ReplicationSet != nil {
+		columns = append(columns, "replication_set")
+		s.ReplicationSet = *labels.ReplicationSet
+	}
+
+	if labels.ExternalGroup != nil {
+		columns = append(columns, "external_group")
+		s.ExternalGroup = *labels.ExternalGroup
+	}
+
+	return columns
+}
```

Comment:
If we really need this function it should be placed in the service_helpers.go file since we have it.